pynsot/commands/cmd_interfaces.py

```diff
@@ -343,6 +343,58 @@ def addresses(ctx, *args, **kwargs):
     )
 
 
+@list.command()
+@click.pass_context
+def parent(ctx, *args, **kwargs):
+    """Get parent of an Interface"""
```

Comment:
Just me nitpicking again... Can you please mirror the similar comments from the Network commands of the same names? I'm just coming from a CLI user-experience perspective!